### DISCUSSION TITLE
Cannot resolve symbol 'GoogleAnalytics' from '@next/third-parties/google'

### DISCUSSION THREAD

**Discussion by mrksph** [vercel/next.js#62058] [Help]
### Summary

Hey there,

I'm using 

```
"@next/third-parties": "^14.1.0",
"next": "^14.1.0",
```

But I still see the error `Cannot resolve symbol 'GoogleAnalytics' from '@next/third-parties/google'` in WebStorm 2023.3.3

<img width="777" alt="image" src="https://github.com/vercel/next.js/assets/10486135/497a0e23-db9b-4881-9268-6e9952da69c1">

Previously I was using Next 13.5.6 then I upgraded to 14 but still facing this issue.

### Additional information

I have deleted node_modules and executed npm install just to make sure all dependencies are updated.

**Comment by avgupta456** [comment]
Also running into this issue

**Comment by isaac-bowen** [comment]
Checkout: https://github.com/vercel/next.js/issues/58697
